### ISSUE TITLE
Re Train

### ISSUE THREAD

**Issue by LIUHAO121** [tensorboy/centerpose#34]
i use coco2017 and your pretrained model dla34.pth to train again,the loss is still growing from 3.1 to round 17.why the loss does keep unchanged or change smaller,

**Comment by gsx1378** [comment]
你好，请问你这个问题解决了吗？我也遇到了相同的问题，训练了30多轮loss基本不变，过程中的评估结果Map也非常低。

**Comment by yzbx** [comment]
> 你好，请问你这个问题解决了吗？我也遇到了相同的问题，训练了30多轮loss基本不变，过程中的评估结果Map也非常低。

我通过修改KeypointHead网络初始化得到缓解，原代码没有初始化self.hm 与 self.hm_hp
`models/heads/keypoints.py`

```
    def init_weights(self):                              
        self.fill_fc_weights(self.wh)
        self.fill_fc_weights(self.hps)
        self.fill_fc_weights(self.reg)
        self.fill_fc_weights(self.hp_offset)
        self.fill_fc_weights(self.hm)
        self.fill_fc_weights(self.hm_hp)
        # https://github.com/princeton-vl/CornerNet/issues/13
        self.hm[-1].bias.data.fill_(-2.19)
        self.hm_hp[-1].bias.data.fill_(-2.19)
```

我只训练了100个epoch, 没有像配置文件中训练1000个epoch, 相应的LR_STEP由[800,900] 改为[80,90]，最后得到如下结果
(COCO上的官方结果为54.5) 

| Backbone | dataset | AP | $AP^{50}$ | $AP^{75}$ | note |
| - | - | - | - | - | - |
| Resnet50 | COCO | 0.460 | 0.759 | 0.475 | - | 
| Resnet50 | CrowdPose | 0.395 | 0.738  | 0.370 | max_objs=32 |
| Resnet50 | CrowdPose | 0.398 | 0.741 | 0.374 | max_objs=64 |
